schemas/payload.json

```diff
@@ -195,9 +195,12 @@
       }
     },
     "env": {
-      "title": "Environment variable mappings.",
+      "title": "Environment variable mappings. Must be strings.",
       "description": "Example: ```\n{\n  \"PATH\": '/borked/path' \n  \"ENV_NAME\": \"VALUE\" \n}\n```",
-      "type": "object"
+      "type": "object",
+      "additionalProperties": {
+        "type": "string"
```

Comment:
Good point. I think we can generate a full gecko task graph, and then inspect it. We could also do the same for NSS.

@jonasfj how much work is it to dig through existing task definitions in Azure and see if any of them have non-strings for env vars? I'm wondering if this is easier than studying all the task graph generators...